CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -163,28 +170,19 @@ public static async Task<KeyPhrasesResult> GetKeyPhrasesAsync(string input, stri
 
             if (!string.IsNullOrEmpty(input))
             {
-                KeyPhraseBatchResult result = await AnalyticsClient.KeyPhrasesAsync(new MultiLanguageBatchInput(
-                    new List<MultiLanguageInput>()
-                    {
-                        new MultiLanguageInput(language, "0", input)
-                    }));
+                ExtractKeyPhrasesResultCollection result = await AnalyticsClient.ExtractKeyPhrasesBatchAsync(
+                    new List<string>() { input }, language);
 
-                if (result.Documents != null)
+                if (!result[0].HasError)
                 {
                     List<string> phrases = new List<string>();
-
-                    foreach (string keyPhrase in result.Documents[0].KeyPhrases)
+                    foreach (string keyPhrase in result[0].KeyPhrases)
                     {
                         phrases.Add(keyPhrase);
                     }
 
                     keyPhrasesResult.KeyPhrases = phrases;
```

Comment:
and same question about why not returning directly what the SDK is returning

Reply:
https://github.com/Azure-Samples/cognitive-services-csharp-call-center/pull/2#discussion_r528565984